canonical-web-and-design/vanilla-framework
# Standardize hover states

## Done

Make buttons, sidenav, contextual menu etc use the same hover color.

## QA

- Pull code
- Run `./run`
Inspect contextual menu, button, side navigation and verify the hover background color is the same.

## Screenshots
![image](https://user-images.githubusercontent.com/2741678/83887152-f39aad00-a73f-11ea-80d6-a82285344314.png)
